org.jacoco.agent.rt.test/src/org/jacoco/agent/rt/internal/output/TcpConnectionTest.java

```diff
@@ -103,8 +104,8 @@ public Void call() throws Exception {
 	 * Remote endpoint is closed before even a valid header was send.
 	 */
 	public void testRemoteCloseWithoutHeader() throws Throwable {
```

Comment:
@marchof a bit unrelated, but I'm wondering why `Test` annotation was removed from this method in - 54e674c0ab96b93668e6c0b036e7773e3ed7ff6d ?